### ISSUE TITLE
[Feature request] Support .xlsb (Excel Binary Workbook)

### ISSUE THREAD

**Issue by Taiizor** [EPPlusSoftware/EPPlus#334]
.xlsx loads 4 times longer than .xlsb and saves 2 times slower and has 1.5 times a bigger file. Reportedly a user from leading forums Stackoverflow tested this on a generated worksheet with 10’000 rows * 1’000 columns = 10’000’000 (10^7) cells of simple chained =…+1 formulas:

<table>
<thead>
<tr>
<th></th>
<th>.xlsx</th>
<th>.xlsb</th>
</tr>
</thead>
<tbody>
<tr>
<td>loading time</td>
<td>165s</td>
<td>43s</td>
</tr>
<tr>
<td>saving time</td>
<td>115s</td>
<td>61s</td>
</tr>
<tr>
<td>file size</td>
<td>91Mb</td>
<td>65Mb</td>
</tr>
</tbody>
</table>

Reference: [Click Me](https://docs.microsoft.com/en-us/openspecs/office_file_formats/ms-xlsb/acc8aa92-1f02-4167-99f5-84f9f676b95a)

**Comment by JanKallman** [comment]
We might look at this long term, but it's not for the coming versions as it requires a totally new format for many of the package parts.

**Comment by RazorTag** [comment]
Has any further consideration been given to support of .xlsb files?

**Comment by JanKallman** [comment]
No, but it's still in our backlog. We will look at what new features for the future in the beginning of next year.
